lib/gui/app/app.js

```diff
@@ -182,15 +182,25 @@ app.run(() => {
     }
 
     const currentFlashState = flashState.getFlashState()
+    const {
+      flashing,
+      verifying,
+      failed
+    } = currentFlashState
```

Comment:
I think we can just use `currentFlashState.flashing` etc. in the following lines; no need to destructure here